app/services/change_schedule.rb

```diff
@@ -131,6 +131,71 @@ def update_ecf_records!
 
     update_school_cohort_and_schedule!
     update_induction_records!
+    update_historical_induction_records!
+  end
+
+  def in_target?(induction_record)
+    in_target_cohort?(induction_record) && in_target_schedule?(induction_record)
+  end
+
+  def in_target_cohort?(induction_record)
+    induction_record.cohort_start_year == new_schedule.cohort_start_year
+  end
+
+  def in_target_schedule?(induction_record)
+    induction_record.schedule == new_schedule
+  end
+
+  def historical_target_school_cohort(historical_record)
+    historical_record.school.school_cohorts.for_year(new_schedule.cohort_start_year).first.presence || SchoolCohort.create!(school: historical_record.school,
```

Comment:
is there a risk that we get a school cohort which is a CIP for a FIP cohort transfer?

Reply:
maybe we can check with Shahad/Nathan around creating school cohorts for schools in different cohorts based on this to be sure, as the amend cohort tool errors on this point?